markdown/assignments/pool_a/skills/index.md

```diff
@@ -7,12 +7,17 @@ weight: 3
 hidden: true
 ---
 
+## Ziel
+
+In dieser Aufgabe sollen Sie verschiedene Fähigkeiten implementieren, die dann vom Spieler verwendet werden können.
 
 ## Fähigkeiten
 
-Entwickeln Sie verschiedene Fähigkeiten die der Spieler im laufe des Spiels freischalten und benutzen kann.
-Überlegen Sie sich ein geeignetes System um die Fähigkeiten auszurüsten, anzuzeigen, freizuschalten und zu benutzen.
-Implementieren Sie mindestens drei unterschiedliche Fähigkeiten, wobei mindestens eine Kampffähigkeit und eine Non-Kampffähigkeit dabei sein muss.
-Eine Fähigkeit soll einen Cooldown bekommen.
-Eine Fähigkeit soll eine Form von Kosten haben (Geld,Leben,Mana o.ä).
-Denken Sie daran: Sie werden vermutlich noch viele unterschiedliche Fähigkeiten einbauen wollen, daher sollten Sie sich ein gutes Konzept für die Implementierung überlegen.
+Entwickeln Sie verschiedene Fähigkeiten, die der Spieler im Laufe des Spiels freischalten und benutzen kann.
+Überlegen Sie sich ein geeignetes System, um die Fähigkeiten auszurüsten, anzuzeigen, freizuschalten und zu benutzen.
+Implementieren Sie mindestens drei unterschiedliche Fähigkeiten, wobei mindestens eine Kampffähigkeit und eine Non-Kampffähigkeit dabei sein muss.
+Außerdem: 
+-    Eine Fähigkeit soll einen Cooldown bekommen.
```

Comment:
Wissen die Studis, was ein "Cooldown" ist?

Reply:
Guter Punkt. Die Frage wird sich vermutlich öfters stellen. 

Meine Meinung: Alles was gängige Spielebegriffe sind, die auch schnell gegooglet werden können, erklären/umschreiben wir nicht.

Richtig glücklich bin ich damit aber nicht...

Reply:
Finde ich schwierig. Wenn Du das auf eine Suche zurückführst, wird die Aufgabe ggf. zu uneindeutig. Wenn dann jemand sagt, ich habe zu dem Begriff "XYZ" die Bedeutung gefunden "blau anmalen" und das implementiert, verweigerst Du ihm dann die Punkte?

Außerdem sollten die Aufgabenstellungen in sich geschlossen sein. 

D.h. wir sollten zumindest eine Idee vermitteln, was wir jeweils von den Studis wollen -- Beispiele angeben o.ä. ...

Reply:
done